jcache/commons/src/test/java/org/infinispan/jcache/AbstractTwoCachesBasicOpsTest.java

```diff
@@ -107,6 +108,21 @@ public void testPutAll(Method m) {
    }
 
    @Test
+   public void testPutAllMapNullValuesNotAllowed(Method m) {
```

Comment:
Also I think it would be great to add 2 simple tests that test to verify NPE is thrown if a null key or value is provided in the map.